conf.py

```diff
@@ -389,6 +389,16 @@ def setup(app):
     app.add_config_value('is_remote_build', None, 'env')  # Whether the build is remotely deployed
     app.add_config_value('source_read_replace_vals', {}, 'env')
     app.connect('source-read', source_read_replace)
+    def upgrade_util_signature_rewrite(app, domain, objtype, contentnode):
```

Comment:
Move it outside the `setup` function, like `source_read_replace`

Reply:
done